src/tlo/methods/cervical_cancer.py

```diff
@@ -0,0 +1,1515 @@
+                sequlae_code=547
+                # "Controlled phase of cervical cancer,Generic uncomplicated disease":
+                # "worry and daily medication,has a chronic disease that requires medication every day and causes some
+                #   worry but minimal interference with daily activities".
+            )
+
+            # For those in stage 4: no palliative care
+            self.daly_wts["stage4"] = self.sim.modules["HealthBurden"].get_daly_weight(
+                sequlae_code=549
+                # "Metastatic phase of cervical cancer:
+                # "Cancer, metastatic","has severe pain, extreme fatigue, weight loss and high anxiety."
+            )
+
+            # For those in stage 4: with palliative care
+            self.daly_wts["stage4_palliative_care"] = self.daly_wts["stage_1_3"]
+            # By assumption, we say that that the weight for those in stage 4 with palliative care is the same as
+            # that for those with stage 1-3 cancers.
+
+        # ----- HSI FOR PALLIATIVE CARE -----
+        on_palliative_care_at_initiation = df.index[df.is_alive & ~pd.isnull(df.ce_date_palliative_care)]
+        for person_id in on_palliative_care_at_initiation:
+            self.sim.modules['HealthSystem'].schedule_hsi_event(
+                hsi_event=HSI_CervicalCancer_PalliativeCare(module=self, person_id=person_id),
+                priority=0,
+                topen=self.sim.date + DateOffset(months=1),
+                tclose=self.sim.date + DateOffset(months=1) + DateOffset(weeks=1)
+            )
+
+    def on_birth(self, mother_id, child_id):
+        """Initialise properties for a newborn individual.
+        :param mother_id: the mother for this child
+        :param child_id: the new child
+        """
+        df = self.sim.population.props
+        df.at[child_id, "ce_hpv_cc_status"] = "none"
+        df.at[child_id, "ce_date_treatment"] = pd.NaT
+        df.at[child_id, "ce_stage_at_which_treatment_given"] = "none"
+        df.at[child_id, "ce_date_diagnosis"] = pd.NaT
+        df.at[child_id, "ce_new_stage_this_month"] = False
+        df.at[child_id, "ce_date_palliative_care"] = pd.NaT
+        df.at[child_id, "ce_date_death"] = pd.NaT
+        df.at[child_id, "ce_date_cin_removal"] = pd.NaT
+        df.at[child_id, "ce_stage_at_diagnosis"] = 'none'
+        df.at[child_id, "ce_ever_treated"] = False
+        df.at[child_id, "ce_cc_ever"] = False
+        df.at[child_id, "ce_xpert_hpv_ever_pos"] = False
+        df.at[child_id, "ce_via_cin_ever_detected"] = False
+        df.at[child_id, "ce_date_cryo"] = pd.NaT
+        df.at[child_id, "ce_current_cc_diagnosed"] = False
+        df.at[child_id, "ce_selected_for_via_this_month"] = False
+        df.at[child_id, "ce_selected_for_xpert_this_month"] = False
+        df.at[child_id, "ce_biopsy"] = False
+
+    def on_hsi_alert(self, person_id, treatment_id):
+        pass
```

Comment:
are you planning on doing something here in the next draft? if not I think we can remove the function.